src/main/java/com/box/sdk/BoxCollaboration.java

```diff
@@ -86,6 +101,9 @@ public BoxCollaboration(BoxAPIConnection api, String id) {
         if (canViewPath != null) {
             requestJSON.add("can_view_path", canViewPath.booleanValue());
         }
+        if (expiresAt != null) {
```

Comment:
This means you can't call this method to clear the expiration, right?  Is there another way to preserve the previous method signature and still allow for clearing the expiration?

Reply:
That's a good point. Just tested out sending null with Postman and the API doesn't accept it. Also tried an empty string and Null and no `expires_at` field. All options do not clear the `expires_at` field. So I think we may not have to have to allow the user to pass in null.
![image](https://user-images.githubusercontent.com/21188938/78720758-40a30400-78f4-11ea-9c1b-19ce6ec392a4.png)